packages/web-components/src/components/va-memorable-date/va-memorable-date.scss

```diff
@@ -41,7 +49,7 @@
 
 :host(:not([uswds])) #dateHint,
 #hint {
-  color: var(--color-gray);
+  color: var(--color-gray-medium);
```

Comment:
This changes the v1 version of hint text. I didn't think this ticket involved making any changes to v1 (or changes to any hint text color at all).

Reply:
Whoops, yeah, must have imagined that one